### PR TITLE
hector gazebo plugin for imu sensor is added

### PR DESCRIPTION
**Pull request by shokoofeh**
Hector Gazebo Plugin for IMU sensor is added. It publishes correct information on /imu/data topic.

### CODE REVIEW

File: jackal_description/urdf/accessories/sick_lms100.urdf.xacro [hunk 1]
```diff
@@ -67,7 +67,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
     <gazebo reference="${prefix}_laser">
       <material>Gazebo/Grey</material>
-      <turnGravityOff>true</turnGravityOff>
+      <turnGravityOff>false</turnGravityOff>
```
Comment (mikepurvis) [inline jackal_description/urdf/accessories/sick_lms100.urdf.xacro:70]: This change should probably be made separately. You can modify the commit to remove it with `git commit --amend`.

File: jackal_description/urdf/jackal.gazebo [hunk 2]
```diff
@@ -6,6 +6,21 @@
     </plugin>
   </gazebo>
 
+  <gazebo>
+    <plugin name="imu_controller" filename="libhector_gazebo_ros_imu.so">
```
Comment (mikepurvis) [inline jackal_description/urdf/jackal.gazebo:10]: There should be a corresponding `package.xml` change to `jackal_gazebo` for this dependency, I assume?

Please also file a bug to raise awareness of this issue in the gazebo upstream: https://github.com/ros-simulation/gazebo_ros_pkgs/issues

**Comment by shokoofeh** [comment]
dependency to hector_gazebo_plugins is added in the same commit for adding imu plugin in jackal.gazebo. 
Also, in a separate commit laser garvity is enabled.

**Comment by mikepurvis** [comment]
Not sure what the convention is about gazebo plugin dependencies— should these be on `jackal_description` or `jackal_gazebo`? I definitely don't want to end up installing Gazebo on the robot's PC.

**Pull request merged**